server/sandbox_run_linux.go

```diff
@@ -375,6 +378,30 @@ func (s *Server) runPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 			}
 			g.SetLinuxCgroupsPath(cgPath + ":" + "crio" + ":" + id)
 			cgroupParent = cgPath
+
+			// check memory limit is greater than the minimum memory limit of 4Mb
+			// expand the cgroup slice path
+			slicePath, err := systemd.ExpandSlice(cgroupParent)
+			if err != nil {
+				return nil, errors.Wrapf(err, "error expanding systemd slice path for %q", cgroupParent)
+			}
+			// read in the memory limit from the memory.limit_in_bytes file
+			fileData, err := ioutil.ReadFile(filepath.Join(cgroupMemoryPath, slicePath, "memory.limit_in_bytes"))
+			if err != nil {
+				return nil, errors.Wrapf(err, "error reading memory.limit_in_bytes file for slice %q", cgroupParent)
+			}
+			// strip off the newline character and convert it to an int
+			strMemory := string(fileData[:len(fileData)-1])
+			if strMemory != "" {
+				memoryLimit, err := strconv.Atoi(strMemory)
+				if err != nil {
+					return nil, errors.Wrapf(err, "error converting from string to int %q", strMemory)
```

Comment:
nit: need more context here that we are hitting this while converting cgroups value

Reply:
Fixed